.github/workflows/release.yml

```diff
@@ -6,13 +6,12 @@ on:
       tag:
         description: "Tag to checkout & release"
         required: true
-  push:
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+"
+  release:
+    types:
+      - published
```

Comment:
I think this should be `released` :
* Published: A release, pre-release, or draft of a release was published.
* Released: A release was published, or a pre-release was changed to a release.
We want to trigger this in releases + pre-releases (for fuji) but not for drafts.

from https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release